src/app/components/books/books.scss

```diff
@@ -0,0 +1,38 @@
+$yellow-star: #EAC435; 
+
+.main-title {
+  padding-left: 7em;
+  h2 {
+    padding-left: 2em;
+  }
+}
+
+.book-item {
+  margin-bottom: 30px;
+}
+
+.caption {
+  p {
+    margin-bottom: 5px;
+  }
+}
+
+.fa-star,
+.fa-star-o {
+  color: $yellow-star;
```

Comment:
create a generic class to set font color to yellow, and then apply this class to the stars. It's not a good practica to assign properties to third party classes, unless you are overriding them